client/components/jetpack/scan-threats/index.tsx

```diff
@@ -125,87 +181,124 @@ const ScanThreats = ( { error, site, threats }: Props ) => {
 							>
-								{ translate(
-									'Auto fix %(fixableCount)s threat',
-									'Auto fix %(fixableCount)s threats',
-									{
-										args: {
-											fixableCount: numberFormat( allFixableThreats.length, 0 ),
-										},
-										comment:
-											'%(fixableCount)s represents the number of auto fixable threats on the site',
-										count: allFixableThreats.length,
-									}
-								) }
+								{ translate( 'Auto fix all' ) }
 							</Button>
 						</>
 					) }
 				</div>
-				{ threats.map( ( threat ) => (
-					<ThreatItem
-						key={ threat.id }
-						threat={ threat }
-						onFixThreat={ () => openDialog( 'fix', threat ) }
-						onIgnoreThreat={ () => openDialog( 'ignore', threat ) }
-						isFixing={ isFixing( threat ) }
-						contactSupportUrl={ contactSupportUrl( site.URL ) }
-						isPlaceholder={ false }
-					/>
-				) ) }
+			</Card>
+			<ThreatListHeader />
+			<div className="scan-threats__threats">
+				{ highSeverityThreats &&
+					highSeverityThreats.map( ( threat ) => (
+						<ThreatItem
+							key={ threat.id }
+							threat={ threat }
+							onFixThreat={ () => openDialog( 'fix', threat ) }
+							onIgnoreThreat={ () => openDialog( 'ignore', threat ) }
+							isFixing={ isFixing( threat ) }
+							contactSupportUrl={ contactSupportUrl( site.URL ) }
+							isPlaceholder={ false }
+						/>
+					) ) }
+				{ ! highSeverityThreats &&
+					lowSeverityThreats &&
+					lowSeverityThreats.map( ( threat ) => (
+						<ThreatItem
+							key={ threat.id }
+							threat={ threat }
+							onFixThreat={ () => openDialog( 'fix', threat ) }
+							onIgnoreThreat={ () => openDialog( 'ignore', threat ) }
+							isFixing={ isFixing( threat ) }
+							contactSupportUrl={ contactSupportUrl( site.URL ) }
+							isPlaceholder={ false }
+						/>
+					) ) }
+				<div class="scan-threats__low-risk">
+					<FoldableCard header={ <ThreatLowRiskItemHeader threatCount={ countMap.low } /> }>
+						{ highSeverityThreats &&
+							lowSeverityThreats &&
+							lowSeverityThreats.map( ( threat ) => (
+								<ThreatItem
+									key={ threat.id }
+									threat={ threat }
+									onFixThreat={ () => openDialog( 'fix', threat ) }
+									onIgnoreThreat={ () => openDialog( 'ignore', threat ) }
+									isFixing={ isFixing( threat ) }
+									contactSupportUrl={ contactSupportUrl( site.URL ) }
+									isPlaceholder={ false }
+								/>
+							) ) }
+					</FoldableCard>
+				</div>
 			</div>
 
 			{ ! error && (
 				<div className="scan-threats__rerun">
 					<p className="scan-threats__rerun-help">
 						{ translate(
-							'If you have manually fixed any of the threats above, you can {{button}}run a scan now{{/button}} or wait for Jetpack to scan your site later today.',
+							'If you have manually fixed any of the threats listed above, you can {{button}}run a manual scan now{{/button}} or wait for Jetpack to scan your site later today.',
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.